### PR TITLE
style: normalize some annotations

### PR DESCRIPTION
**Pull request by healthjyk**
#### What type of PR is this?
cleanup

#### What this PR does / why we need it:
normalize annotations

#### Special notes for your reviewer:
None

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., design docs, usage docs, etc.:

```docs
None
```

### CODE REVIEW

File: zh_CN.po~ [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (SparkYuan) [inline zh_CN.po~:1]: what is this file used for?

Comment (healthjyk) [reply zh_CN.po~:1]: Sorry, I'll delete it

**Comment by adohe** [comment]
please squash before we can merge.

**Review by SparkYuan — APPROVED** [review]
lgtm

**Pull request merged**